samples/BikeSharingApp/README.md

```diff
@@ -1,4 +1,79 @@
-# Bike Sharing Sample: Iteratively Develop and Debug Microservices in Kubernetes
+# Deploy the Bike Sharing sample application to Azure Kubernetes Service
 
-## Quickstart
-[Quickstart: Team development on Kubernetes using Azure Dev Spaces](https://docs.microsoft.com/en-us/azure/dev-spaces/quickstart-team-development).
\ No newline at end of file
+*Bike Sharing* is a microservices-based sample application that helps showcase the sandboxing capabilities of [Azure Dev Spaces](https://aka.ms/devspaces). 
+
+Follow the steps below to deploy this sample app to Azure Kubernete Service (AKS).
+
+## Prerequisites
+* [Azure subscription](https://azure.microsoft.com/free)
+* [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest)
+* [Helm 2.13 (or greater)](https://github.com/helm/helm/blob/master/docs/install.md)
+
+## Configure Azure resources
+
+1. **Create an Azure Kubernetes Service cluster.** You must create an AKS cluster in a [supported region](https://docs.microsoft.com/azure/dev-spaces/about#supported-regions-and-configurations). The below commands create a resource group called `MyResourceGroup` and an AKS cluster called `MyAKS` in the `eastus` region.
+
+    ```bash
+    az login
+    az group create --name MyResourceGroup --location eastus
+    az aks create -g MyResourceGroup -n MyAKS --location eastus --disable-rbac --generate-ssh-keys
+    ```
+
+1. **Enable Azure Dev Spaces on the AKS cluster.**
+    ```bash
+    az aks use-dev-spaces -g MyResourceGroup -n MyAKS --space master --yes
+    ```
+
+## Deploy the BikeSharing sample app
+
+1. **Create a fork of this GitHub repo.** It's better to create a *fork* for this exercise so that you can save your own commits and repo configuration.
```

Comment:
>It's better to create a *fork* for this exercise so that you can save your own commits and repo configuration. [](start = 42, length = 110)

Nit: please consider adding something like "Use the GitHub menu to fork this repository.". This quickstart is built on a "Description + Action" pattern, where first we explain what we're going to do and then how to do it.
When following this quickstart, I thought something was wrong there because the "how to do it" was missing for this fork case.
Anyway that's not a big deal. :)

Reply:
I added a link to GitHub documentation on how to create a fork.